components/header.js

```diff
@@ -13,6 +13,7 @@ export default class Header extends React.Component {
           <meta charSet='utf-8'/>
           <meta name='viewport' content='initial-scale=1.0, width=device-width'/>
           <meta name='description' content={description} />
+          <link rel="alternate" title="Events Detected by OONI" href="http://fastpath.ooni.nu:8080/rss/global.xml" type="application/rss+xml" />
```

Comment:
This should rather be: https://explorer.ooni.org/rss/global.xml

Reply:
This PR wasn't ready for review yet :grin: